packages/cds-odata-annotation-converter/test/setup.ts

```diff
@@ -64,26 +64,19 @@ export const getPaths = getDeserializer<string[]>('paths.json');
 
 export const prepare = async (
     projectRootFolder: string,
-    cdsServiceName: string,
     additionalFilesToLoad: string[] = []
-): Promise<{
-    projectRoot: string;
-    cdsArtifacts: CdsArtifactsType;
-    fileCache: Map<string, File>;
-}> => {
+): Promise<CdsCompilerFacade> => {
     const projectRoot: string = projectRootFolder;
     const roots = await projectAccess.getCdsRoots(projectRoot);
-    const resolvedRoots = cds.resolve(roots ?? []);
-    const cdsFiles = resolvedRoots?.map((uri) => relative(projectRoot, uri));
-    let fileCache: Map<string, File> = new Map();
+    const cdsFiles = (roots ?? [])?.map((uri) => relative(projectRoot, uri));
```

Comment:
This can be further simplified to, because we do not actually need to provide cached files in this case.
```
const files = await getCdsFiles(projectRootFolder);
return createCdsCompilerFacadeForRoot(projectRootFolder, files);
```

Reply:
done